packages/components/src/ui/color-picker/styles.ts

```diff
@@ -53,9 +91,26 @@ export const ColorfulWrapper = styled.div`
 	.react-colorful__pointer {
 		height: 16px;
 		width: 16px;
+		border: ${ CONFIG.borderWidthFocus } solid rgba( 255, 255, 255, 0 );
+		box-shadow: inset 0px 0px 0px ${ CONFIG.borderWidthFocus } #ffffff;
```

Comment:
We should probably use the `box-shadow` technique for borders, as suggested by @jasmussen in https://github.com/WordPress/gutenberg/pull/34712#discussion_r713728714

Reply:
I'm not sure if this technique works for this case given that we need border and box-shadow at the same time.

Reply:
You can stack box shadows, that's what I did in https://github.com/WordPress/gutenberg/commit/2997a2f2b933b6d157862bf7ad50a67411abea79.